src/sentry/search/events/fields.py

```diff
@@ -361,6 +361,12 @@ def parse_arguments(function, columns):
     return [arg for arg in args if arg]
 
 
+def to_tuple(x):
```

Comment:
thoughts on something more descriptive as the name? My first impression of `to_tuple` was that it was only a `tuple` cast

Maybe something like ... `arguments_to_tuple`? (only slightly better though 😞)

Reply:
How about `format_column_as_key`? Since the intent of this function is to format a column value to be hashable so it can be used as a key to the dict.

Reply:
:+1: Much better